files/ja/web/javascript/reference/errors/stmt_after_return/index.md

```diff
@@ -43,13 +43,13 @@ unreachable code after a return statement は、以下のような場合に発
 function f() {
   var x = 3;
   x += 4;
-  return x;   // return は直ちに関数を終了します。
-  x -= 3;     // そのため、この行は実行されません。到達不可能です。
+  return x; // return は直ちに関数を終了します。
+  x -= 3; // そのため、この行は実行されません。到達不可能です。
 }
 
 function f() {
-  return     // これは `return;` のように扱われます。
-    3 + 4;   // そのため、関数を抜け出し、この行には決して到達しません。
+  return; // これは `return;` のように扱われます。
```

Comment:
This `return` is intentionally written without a semicolon, as a bad example, so a semicolon shouldn't be added.